src/WebJobs.Script.WebHost/Management/MeshServiceClient.cs

```diff
@@ -7,9 +7,9 @@
 using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
```

Comment:
@balag0 I had to use the Azure.Common dependency (still using CloudStorageAccount) to maintain the functionality here. It would be good to flag this to use the latest SDKs